pennylane/math/single_dispatch.py

```diff
@@ -235,6 +272,15 @@ def _scatter_element_add_tf(tensor, index, value):
 ar.register_function("torch", "shape", lambda x: tuple(x.shape))
 ar.register_function("torch", "gather", lambda x, indices: x[indices])
 
+ar.register_function(
+    "torch",
+    "fft.ifft2",
+    lambda a, s=None, axes=(-2, -1), norm=None: _i("torch").fft.ifft2(
+        input=a, s=s, dim=axes, norm=norm
+    ),
+)

```

Comment:
The `torch.fft.ifft2` functionality matches almost exactly with the NumPy functionality.

However, it uses a slightly different signature - instead of `a` and `axes`, it uses `input` and `dim`! Here we use `register_function` with a very light lambda function in order to register the different named parameter arguments.

This catches the case where a user might pass keyword arguments, `qml.math.fft.ifft2(tensor, axes=...)`.